app/assets/stylesheets/app/dashboard.scss

```diff
@@ -47,36 +43,70 @@
 
   &--head {
     @extend .matrix__user;
+    height: 55px;
     flex: 0 0 65px;
     padding: 0;
-    padding-top: 12px;
   }
 }
 
 .matrix__user-img {
+  display: block;
   max-height: 48px;
   max-width: 48px;
   border-radius: 50%;
   border: 1px solid $secondary-border;
+
+  &--first {
+    @extend .matrix__user-img;
+    border: 3px solid $first-img;
+  }
+
+  &--second {
+    @extend .matrix__user-img;
+    border: 3px solid $second-img;
+  }
+
+  &--third {
+    @extend .matrix__user-img;
+    border: 3px solid $third-img;
+  }
+}
+
+.matrix__user-crown {
+  z-index: 1;
+  height: 48px;
+  width: 48px;
+}
+
+.matrix__user-div {
+  display: inline-block;
+  position: relative;
+  z-index: 1;
+}
+
+.matrix__user-div + .matrix__user-crown {
+  position: absolute;
+  z-index: 1;
+  top: -16px;
+  right: -10px;
 }
 
 .matrix__picture {
   text-align: center;
 }
 
 .matrix__pullrequest {
-  background-color: $white;
+  width: 66px;
   border: .5px solid $primary-border;
-  display: flex;
-  flex: 0 0 66px;
-  flex-direction: column;
-  height: 69px;
-  justify-content: center;
-  margin: -1px 0 0 -1px;
+  background-color: $white;
   padding: 0;
+  margin: -1px 0 0 -1px;
```

Comment:
Shorthands of length `4` are not allowed. Value was `-1px 0 0 -1px`

Reply:
El mono está alegando por cosas que estaban antes y contra algunas convenciones de BEM...